ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -9,6 +9,22 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 
 {-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+-- | Property tests for the mempool.
+--
+-- The mempool collects transactions from down-stream nodes, makes them
+-- available to up-stream nodes, and of course provides the pool of transactions
+-- that we use when forging blocks.
+--
+-- These tests for the mempool are not model based, but instead check various
+-- simple properties and invariants:
+--
+-- * After adding valid transactions to the mempool, they can be retrieved.
+-- * Adding invalid transactions from the mempool will report them as invalid,
+--   and they are not added.
+-- * Transactions cannot be retrieved after they are removed.
+-- * The mempool capacity is not exceeded
+-- * etc.
```

Comment:
I would instead say that they "check simple properties like for example:" instead of having an "etc" last bullet point

Reply:
Good point. Done.